drape/text_itemizer.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <harfbuzz/hb.h>
+
+namespace text_itemizer
```

Comment:
@vng any ideas on better names for namespace and classes?

Reply:
As you wish

Reply:
Renamed.